cmd/bgpmon/cmd/root.go

```diff
@@ -79,7 +79,7 @@ func Execute() {
 func init() {
 	cobra.OnInitialize(initConfig)
 	rootCmd.PersistentFlags().StringVarP(&cfgFile, "config", "c", "$HOME/.bgpmon.yaml", "Config file")
-	rootCmd.PersistentFlags().StringVarP(&bgpmondHost, "host", "h", "127.0.0.1", "bgpmond host to connect to")
+	rootCmd.PersistentFlags().StringVarP(&bgpmondHost, "host", "H", "127.0.0.1", "bgpmond host to connect to")
```

Comment:
What's the reason for this change?

Reply:
The cobra library panics when you try to override the "h" flag, because it considers it pre-defined as the help flag. To maintain consistency, we were going to change all of the global flags to be capitalized.

Reply:
Sounds good to me.